apps/examples/package.json

```diff
@@ -29,8 +29,8 @@
 		"dev": "vite --host",
 		"build": "vite build",
 		"lint": "yarn run -T tsx ../../scripts/lint.ts",
-		"e2e": "playwright test -c ./e2e/playwright.config.ts",
-		"e2e-ui": "playwright test --ui -c ./e2e/playwright.config.ts"
+		"e2e": "NODE_ENV=test && playwright test -c ./e2e/playwright.config.ts",
```

Comment:
Fixes [some tests that started failing](https://github.com/tldraw/tldraw/actions/runs/8158802282/job/22301557609) as a result of these changes. This [makes stuff run sync](https://github.com/tldraw/tldraw/blob/mitja%2Fraf/packages/utils/src/lib/throttle.ts#L3) instead of on the next frame.